src/components/FilmList/fixture.ts

```diff
@@ -0,0 +1,41 @@
+const data: any = {

```

Comment:
You defined the interface "FilmDetails", su you should use it here.
`const data: FilmDetails = { ... }`